### PR TITLE
Add nodejs package

### PR DESCRIPTION
**Pull request by skattoju4**
nodejs is needed to build the new ui. I figured it would be as easy as adding nodejs to the list of packages installed in the docker container. Not sure if the tag needs to be updated ?

### CODE REVIEW

**Comment by khos2ow** [comment]
Thank you Sid, let me fix the CI and I'll get back to this shortly.

File: centos6/Dockerfile [hunk 1]
```diff
@@ -48,6 +48,8 @@ RUN \
     yum clean metadata && \
     yum install -y \
         apache-maven && \
+    curl -sL https://rpm.nodesource.com/setup_12.x | bash - && \
```
Comment (khos2ow) [inline centos6/Dockerfile]: CentOS 6 has reached end of life and this is not really needed anymore.

**Review by khos2ow — COMMENTED** [review]
@skattoju4 please rebase on master and check my one comment below.

File: centos6/Dockerfile (hunk 1 above)
Comment (skattoju4) [reply centos6/Dockerfile]: ive removed my change from the centos6 Dockerfile

**Comment by skattoju4** [comment]
sorry about the force pushes i keep missing the sign-offs

**Review by khos2ow — COMMENTED** [review]
Add `nodejs` to "Packages installed in container" section in readme as well, otherwise, pending CI status, lgtm.

**Comment by khos2ow** [comment]
@skattoju4 I think the failure in `centos7-jdk11` is not related to this change, can you double check please? And if it's not related, can you sqaush the commits?

**Comment by skattoju4** [comment]
> @skattoju4 I think the failure in `centos7-jdk11` is not related to this change, can you double check please? And if it's not related, can you sqaush the commits?

@khos2ow It fails because it is unable to download some jar, unrelated to my changes. How do i squash commits ? I thought thats an option you specify when merging ?

**Comment by skattoju4** [comment]
> > @skattoju4 I think the failure in `centos7-jdk11` is not related to this change, can you double check please? And if it's not related, can you sqaush the commits?
> 
> @khos2ow It fails because it is unable to download some jar, unrelated to my changes. How do i squash commits ? I thought thats an option you specify when merging ?

never mind commits squashed.

**Pull request merged**